builtin_object.go

```diff
@@ -473,6 +473,9 @@ func (r *Runtime) objectproto_toString(call FunctionCall) Value {
 			clsName = classArray
 		} else {
 			clsName = obj.self.className()
+			if clsName == classBigInt {
```

Comment:
Are there any places where the class name needs to be classBigInt? It's largely a leftover from ES5, since then it's mostly covered by Symbol.toStringTag, I'm not even sure it's required anymore...

Reply:
In the TC39 tests, the result of BigInt is [object Object].

Reply:
What meant was, getting rid of `classBigInt` completely. The only case where it's used is for `*objectGoReflect`, but it could be changed to checking against `exportType()`.